control/proto/gateway.proto

```diff
@@ -236,12 +276,47 @@ message nsid_status {
 }
 
 message subsystems_info {
+        repeated subsystem subsystems = 1;
```

Comment:
I really don't like duplicating all the message types and I don't really understand why is this needed. Just to save you some conversion calls.

Reply:
-list namespaces
-list listeners
- list subsystem  
all those use the same spdk get subsystems JSON RPC call internally. We could expose a singe get_subsystems GRPC, considering minimal interface, and move list namespaces, list listeners and list subsystem logic from grpc server to client cli side. wdyt?

Reply:
sorting and formatting of the data sounds a client side logic to me, I would strive to a minimal interface.